app/controllers/sessions_controller.rb

```diff
@@ -11,6 +11,7 @@ def create
       setup_webauthn_authentication
       setup_mfa_authentication
 
+      session[:mfa_created_at] = Time.now.utc
```

Comment:
I'm a little unsure about the name `mfa_created_at`; I'd probably wonder if it was tied to a model. Could it be something like `mfa_login_started_at`?

Reply:
That's a fair point. I like your suggestion, i'll update 👍!